script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  mobileMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.mobile-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  mobileMenu.classList.remove('active');
+}));
```

Comment:
- [ ] Very nice implementation :clap: you did a nice job  :+1: but your code is not `DRY` Kindly write code with `DRY` principles. `DRY` stand for `Don't Repeat Yourself`, a fundamental principle of software development aimed at reducing the repetition of information. Check below :point_down: my suggestions :slightly_smiling_face: 

 ```suggestion
const hamburger = document.querySelector('.hamburger');
const mobileMenu = document.querySelector('.mobile-menu');
const mobileLinks = document.querySelectorAll('.mobile-link');

const toggleActiveClass = (element) => {
  element.classList.toggle('active');
};

const removeActiveClass = (element) => {
  element.classList.remove('active');
};


hamburger.addEventListener('click', () => {
  toggleActiveClass(hamburger);
  toggleActiveClass(mobileMenu);
});

mobileLinks.forEach((link) => {
  link.addEventListener('click', () => {
    removeActiveClass(hamburger);
    removeActiveClass(mobileMenu);
  });
});
```

- [ ] In this refactored code, we extracted the common functionality of toggling and removing the active class into separate functions `toggleActiveClass` and `removeActiveClass`. These functions take an element as a parameter and perform the corresponding action.

- [ ] We then use these functions in the event listeners for the `hamburger` and `mobileLinks` elements, avoiding the repetition of the code. We also assigned the `mobileLinks` query result to a variable for better code readability and maintenance.